apps/container-registry/internal/app/graph/struct-to-graphql/build.graphqls

```diff
@@ -10,10 +8,9 @@ type Build @shareable {
   markedForDeletion: Boolean
   name: String!
   recordVersion: Int!
-  repository: String!
   source: Kloudlite_io__apps__container___registry__internal__domain__entities_GitSource!
+  spec: Github_com__kloudlite__operator__apis__distribution__v1_BuildRunSpec!
```

Comment:
re run `task k8s-schema`, the naming pattern has actually been updated

now, this `Kloudlite_io__apps__container___registry__internal__domain__entities_GitSource` would be updated into this `Kloudlite__io___apps___container____registry___internal___domain___entities__GitSource`

tldr: one more underscore in everything


then, issue `task gql`, and then also fix the resolvers

Reply:
Fixed